src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -82,6 +82,27 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
```

Comment:
Please move default config with example to be first

Please read Attention section at https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-examples